code/jeff/Python/oop_rectangle.py

```diff
@@ -0,0 +1,28 @@
+
+# define class
+class Rectancle:
+    def __init__(self, length, width):
+        self.length = length
+        self.width = width
+
+    #perimeter method
+    def perimeter(self):
+        return (self.length + self.width) * 2
+
+    def area(self):
+        return self.length * self.width
+
+        
+rectangle_1 = Rectancle(1,2)
+rectangle_2 = Rectancle(30,5)
+rectangle_3 = Rectancle(2,10)
+rectangle_4 = Rectancle(10,25)
+
+print(rectangle_3.perimeter())
+
```

Comment:
Missing the implementation of the Parallelepipede child class  . Lab description [here](https://github.com/PdxCodeGuild/class_chihuahua/blob/main/1%20Python/labs/OOP.md)